lib/class-wp-rest-pattern-directory-controller.php

```diff
@@ -183,7 +183,7 @@ public function prepare_item_for_response( $raw_pattern, $request ) { // phpcs:i
 		 * @param object           $raw_pattern The unprepared pattern.
 		 * @param WP_REST_Request  $request     The request object.
 		 */
-		return apply_filters( 'rest_prepare_application_password', $response, $raw_pattern, $request );
+		return apply_filters( 'rest_prepare_block_pattern', $response, $raw_pattern, $request );
```

Comment:
Was this a bug from before?

Reply:
Yeah, it was found in the core PR: https://github.com/WordPress/wordpress-develop/pull/1298#discussion_r639139371 - sounds like it was copied and not updated to the correct filter.